### ISSUE TITLE
PhpBrowser and Symfony redirects generating incorrect routes

### ISSUE THREAD

**Issue by benr77** [Codeception/Codeception#2589]
I have a test suite using PhpBrowser to test my Symfony2 app using a particular environment for Codeception. Codeception is 2.1.1

Here is the suite.yml file

```
class_name: HttpCacheTester
modules:
  enabled:
    - PhpBrowser:
        url: http://my.application.com/app_dev.php/
    - \Helper\HttpCache
    - Asserts
```

If I write acceptance tests for this, they work ok, except for when I am submitting a form. 

The problem lies with the Symfony2 generateURL() method used when redirecting after a form submission. It produces a URL that includes the app_dev.php prefix. In my case `/app_dev.php/index_route`. 

Normally this would be ok as it's root relative, but PhpBrowser already has a base URL from it's configuration that contains this, so the final URL that PhpBrowser tries to access is 

`http://my.application.com/app_dev.php/app_dev.php/index_route`

This then results in a route not found error obviously.

Any suggestions would be appreciated.

Thanks

**Comment by benr77** [comment]
It appears that this issue is fixed by upgrading Codeception to 2.1.4 and PHPUnit to 4.8.18 etc. My tests now complete as expected. Hopefully this could be helpful for someone else in the future.

**Comment by DavertMik** [comment]
Thanks for sharing
